libs/ui/src/search_select.tsx

```diff
@@ -0,0 +1,342 @@
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import { assert, assertDefined } from '@votingworks/basics';
+import { Icons } from './icons';
+
+function noop() {
+  // do nothing
+}
+
+export interface SelectOption<T extends string = string> {
+  value: T;
+  label: string;
+}
+
+const ComponentContainer = styled.div`
+  width: 100%;
+  position: relative;
+`;
+
+const SelectBar = styled.div`
+  width: 100%;
+  min-height: 2.2rem;
```

Comment:
I just manually adjusted this so that, in multi-select mode, the addition of options wouldn't cause the box's size to noticeably change. It can fit a single row of options.